lib/Doctrine/Migrations/Configuration/Migration/ConfigurationArray.php

```diff
@@ -61,6 +61,9 @@ public function getConfiguration(): Configuration
             'all_or_nothing' => static function ($value, Configuration $configuration): void {
                 $configuration->setAllOrNothing(is_bool($value) ? $value : BooleanStringFormatter::toBoolean($value, false));
             },
+            'transactional' => static function ($value, Configuration $configuration): void {
+                $configuration->setAllOrNothing(is_bool($value) ? $value : BooleanStringFormatter::toBoolean($value, true));
```

Comment:
Why don't we use the "setTransactional" setter here?
```php
$configuration->setTransactional(is_bool($value) ? $value : BooleanStringFormatter::toBoolean($value, true));
```

Reply:
I think it was a copy/paste mistake